cpp/tests/binaryop/binop-compiled-fixed_point-test.cpp

```diff
@@ -684,6 +684,122 @@ TYPED_TEST(FixedPointCompiledTestBothReps, FixedPointBinaryOpThrows)
                cudf::logic_error);
 }
 
+TYPED_TEST(FixedPointCompiledTestAllReps, FixedPointBinaryOpModSimple)
```

Comment:
Are there tests for the case where the scales are not equal? If not, we need them.